libraries/rib-base/src/main/java/com/badoo/ribs/core/Node.kt

```diff
@@ -48,6 +49,7 @@ open class Node<V : RibView> @VisibleForTesting internal constructor(
     val buildParams: BuildParams<*>,
     private val viewFactory: ViewFactory<V>?,
     private val retainedInstanceStore: RetainedInstanceStore,
+    private val isOnMainThreadFunc: () -> Boolean,
```

Comment:
We need to make it overridable for every junit tested Node in our codebase, which does not seem achievable.

In general makes sense (we do not support any multi-threading anyway, comparing to Appyx which is guarded by coroutines context and/or atomic mutable states).

Let's take a default implementation from `RIBs` singleton and add `RIBs.isOnMainThread` and try to override it somehow in tests, potentially in NodeTestHelper and InteractorTestHelper? But the problem there that we do not reset them. (Maybe it is ok just to give up and override always without reset).